src/hal/src/image.rs

```diff
@@ -589,7 +589,14 @@ pub struct SubresourceLayers {
     pub layers: Range<Layer>,
 }
 
-/// A subset of resources contained within an image.
+/// When creating an image view, used for describing the view's purpose and which part
```

Comment:
I'm not sure if this is the appropiate documentation for `SubresourceRange`. This rather fits for `create_image_view`.

Reply:
Are SubresourceRanges used for anything other than describing details when creating image views?